crates/conformance-test-utils/src/main.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Parser;
+use conformance_test_utils::utils::compute_starknet_address;
+use starknet::core::types::FieldElement;
+
+#[derive(Parser)]
+pub struct ComputeStarknetAddressArgs {
+    #[arg(long)]
+    #[arg(short = 'k')]
+    #[arg(help = "Set the kakarot address.")]
+    pub kakarot_address: FieldElement,
+    #[arg(long)]
+    #[arg(short = 'a')]
+    #[arg(help = "Set the account proxy class hash.")]
+    pub account_proxy_class_hash: FieldElement,
+    #[arg(long)]
+    #[arg(short = 'e')]
+    #[arg(help = "Set the EVM address.")]
+    evm_address: FieldElement,
+}
+
+fn main() {
+    let args = ComputeStarknetAddressArgs::parse();
+
+    // Compute StarkNet address
+    let starknet_address =
+        compute_starknet_address(args.kakarot_address, args.account_proxy_class_hash, args.evm_address);
+
+    // Convert FieldElement to Hex String
+    let starknet_address: Vec<String> =
+        starknet_address.to_bytes_be().iter().map(|byte| format!("{:02x}", byte)).collect();
+    let starknet_address = starknet_address.join("");
```

Comment:
There's no `to_hex_be` for FieldElement so I did this for now

Reply:
@ftupas you can use the [`UfeHex`](https://github.com/xJonathanLEI/starknet-rs/blob/master/starknet-core/src/serde/unsigned_field_element.rs) from starknet-rs if you want, I think that does what you are looking for (you might need to wrap your FieldElement inside a struct and use `serde_with` as [here](https://github.com/xJonathanLEI/starknet-rs/blob/master/starknet-providers/src/jsonrpc/mod.rs#L147)).

Reply:
Thanks! I'll use that instead 🙏